rotkehlchen/api/rest.py

```diff
@@ -1314,12 +1314,40 @@ def user_premium_key_remove(self) -> Response:
         # else
         return api_response(OK_RESULT, status_code=HTTPStatus.OK)
 
-    def query_all_assets(self, filter_query: AssetsFilterQuery) -> Response:
+    def query_all_assets(
+            self,
+            include_ignored_assets: bool,
+            show_user_owned_assets_only: bool,
+            **kwargs: Any,
```

Comment:
Okay so now I see why you did kwargs here, but this is changing the way we do stuff a bit and it's not needed. You can still do this all in the schema.

You can pass the user db in the schema itself and then you can get `ignored_assets_identifier` from there, get `user_owned_assets_identifiers` from there and then create the `AssetsFilterQuery` in the schema.

For example here the `ethereum_manager` is passed as an argument in the schema: https://github.com/rotki/rotki/blob/172c1ad9d055200eeeeb6dd93e4f063739f9b9e8/rotkehlchen/api/v1/schemas.py#L1548

you need to do something similar with the user db to have it doneright.